viewer/api/views.py

```diff
@@ -77,3 +77,83 @@ def metric(request, metric_id):
 
     serializer = DistributionSerializer(qs, populations=pops, subgroup=sg)
     return Response(serializer.data)
+
+
+# API v2
+
+@api_view(['GET'])
+def experiments(request):
+    datasets = (
+        DataSet.objects.visible()
+                       .order_by(F('created_at').desc(nulls_last=True))
+    )
+    data = []
+    for d in datasets:
+        data.append({
+            'id': d.id,
+            'name': d.name,
+            'completed': True,  # TODO
```

Comment:
In the meantime, maybe `False` would be a better default. I'd rather people think that a completed experiment is ongoing than vice-versa.

Is it feasible to get an accurate value for this at all from upstream?

Reply:
One idea... We will eventually stop receiving pings as the experiment wraps up. During import I can set all experiments as completed, then for those experiments where I do update the data, I set this flag to False b/c we are still receiving data.

Reply:
That sounds great. I'll open another issue for that.